### DISCUSSION TITLE
Top and bottom toolbar alignments with mantine v7 and AppShell

### DISCUSSION THREAD

**Discussion by logycon** [KevinVandy/mantine-react-table#366] [General]
First, thank you for this component, it's outstanding.

Question:

What could cause top (search, etc) and bottom (pagination) toolbars to align left instead of right? I am using latest mantine 7.11, with AppShell and latest next. 

![image](https://github.com/KevinVandy/mantine-react-table/assets/538881/bfea9785-b163-4b15-8bf1-800977af0997)

Thanks in advance!

**Comment by logycon** [comment]
Bottom toolbar:

![image](https://github.com/KevinVandy/mantine-react-table/assets/538881/b993a8ad-e6a2-4bb3-8dd6-e2a94e6c0837)

**Comment by KevinVandy** [comment]
Maybe wrong CSS import order?

**Reply by logycon** [reply]
Thanks! That was it. Always the little forgetful things that get me.
